lib/core/src/irods_error.cpp

```diff
@@ -136,6 +136,11 @@ const char *error::status_token_     = " status [";
         std::string  _fcn,
         const error& _rhs ) :
         error(_rhs) {
+ 
+        if (_msg != "") {
+            message_ = _msg;
```

Comment:
You can possibly use `std::move` here given that `_msg` arrives as a copy.

Reply:
Could not use std::move since _msg is being used below this point.